src/adapter/src/optimize/materialized_view.rs

```diff
@@ -89,8 +89,8 @@ impl Optimizer {
             typecheck_ctx: empty_context(),
             catalog,
             compute_instance,
-            exported_sink_id,
-            internal_view_id,
+            sink_id: exported_sink_id,
+            view_id: internal_view_id,
```

Comment:
Can you rename the parameters to `sink_id` and `view_id` as well, for consistency?

Reply:
Will do.